client/src/PoemBox.jsx

```diff
@@ -44,7 +44,8 @@ class PoemBox extends React.Component {
       url: this.props.url,
       dataType: 'json',
       type: 'POST',
-      data: poem,
+      contentType: 'application/json',
```

Comment:
The recommendation seems to be `application/json; charset=utf-8`. Let's do that here and in `savePoems`.